src/daemon/profile.c

```diff
@@ -62,9 +63,9 @@ void urlencode2(char* dst, const char* src){
 
 // GUID conversions
 int setid(usbid* id, const char* guid){
-    int32_t data1;
-    int16_t data2, data3, data4a;
-    char data4b[6];
+    uint32_t data1;
+    uint16_t data2, data3, data4a;
+    unsigned char data4b[6];
```

Comment:
For consistency, please use the typedefs when possible.
https://github.com/ckb-next/ckb-next/blob/master/src/daemon/includes.h#L24-L26